resources/sdk/pureclouddotnet/templates/ApiClient.mustache

```diff
@@ -113,7 +113,7 @@ namespace {{packageName}}.Client
         /// Gets or sets the RestClient.
         /// </summary>
         /// <value>An instance of the RestClient</value>
-        public RestClient RestClient { get; set; }
+        private RestClient RestClient { get; set; }
```

Comment:
Don't change this. This property is public so consumers can directly interface with the rest client object when necessary. This will be a breaking change for SDK users and is likely to come back to bite you as someone is probably using it.

Reply:
Want to mention about this comment, Though it is not about this PR. We did a RestSharp upgrade recently and with this RestClient Class has become more of a wrapper and all its impl methods are moved to a different class. Making this private will also ensure the client can no longer use this and have to shift to the clientOptions.  Had a blogpost as well for the upgrade changes.